flask_restful/__init__.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, app=None, prefix='',
                  default_mediatype='application/json', decorators=None,
                  catch_all_404s=False, serve_challenge_on_401=False,
                  url_part_order='bae', errors=None):
-        self.representations = dict(DEFAULT_REPRESENTATIONS)
+        self.representations = OrderedDict(DEFAULT_REPRESENTATIONS)
```

Comment:
`DEFAULT_REPRESENTATIONS` should be declared as a `DefaultDict` instead of converted here:

```python
DEFAULT_REPRESENTATIONS = OrderedDict([('application/json', output_json)])
```

Otherwise, if more items are ever added to it, it would have the same potentially arbitrary ordering that you were trying to fix with this PR.